### ISSUE TITLE
Get started guide for newbies

### ISSUE THREAD

**Issue by aawiseman** [maproulette/maproulette2#84]
As someone who only worked on a few MapRoulette tasks on the old version on the iD editor side and never generated them myself, I think a little getting started guide would be helpful: here's what to paste and where, how long it takes to load, here's how to run the new task once it's created, and so on.

**Comment by mvexel** [comment]
A video perhaps?

**Comment by Kopkins** [comment]
I will be working on creating tasks and challenges in the near future. I was thinking that I might be able to contribute some documentation on how to get challenges up and running, how to add/manage tasks, etc. This may be something I can help with

**Comment by mvexel** [comment]
Yes that would be fantastic! Thanks. Looking forward to your challenges as well.

Bear in mind that the UI for creating challenges will change a bit still, so it may be wise to wait a little if you are going that route. The API should be fairly stable by now however.

**Comment by reichg** [comment]
Is the learning section enough for this right now?
